utils/issue-format-bot/index.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 AJ Jordan
+//
+// This file is part of the HTTPS Everywhere issue management bot.
+//
+// The HTTPS Everywhere issue management bot is free software: you can
+// redistribute it and/or modify it under the terms of the GNU Affero
+// General Public License as published by the Free Software
+// Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// The HTTPS Everywhere issue management bot is distributed in the
+// hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+// the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+// PURPOSE.  See the GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with the HTTPS Everywhere issue management bot.  If
+// not, see <http://www.gnu.org/licenses/>.
```

Comment:
You can remove this header from the files - as long as you have a `LICENSE` file, this is already clear.  And since there are a lot of files with this, it'll save us some space.  If you want, you can consolidate this as:

```js
// Copyright 2017 AJ Jordan, AGPLv3+
```